.github/workflows/deploy-docs.yml

```
@@ -0,0 +1,191 @@
+          GITHUB_TOKEN: ${{ secrets.CONSENT_GIT_TOKEN }}
+        with:
+          header: c15t-docs-preview
+          number: ${{ steps.find_pr.outputs.pr_number }}
+          message: |
+            🚀 Your documentation preview is ready!
+
+            🔗 Live URL: ${{ steps.deploy.outputs.url }}
+
+            🧪 This sticky comment auto-updates on every push.
+            📦 Source: branch `${{ github.ref_name }}` @ `${{ github.sha }}`
+            🧭 Trigger: `${{ github.event_name }}`
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)